.github/label-pr-config.yml

```diff
@@ -80,13 +80,13 @@ subSystemLabels:
   /^deps\/nghttp2\/nghttp2\.gyp/: build, http2
   /^deps\/nghttp2\//: http2
   /^deps\/ngtcp2\//: quic
-  /^deps\/([^/]+)/: dependencies, $1
+  /^deps\/([^\/]+)/: dependencies, $1
 
   ## JS subsystems
   # Oddities first
   /^lib\/(?:punycode|\w+\/freelist|sys\.js)/: ''
   /^lib\/constants\.js$/: lib / src
-  /^lib\/internal/debugger$/: debugger
+  /^lib\/internal\/debugger$/: debugger
```

Comment:
I think this is the only change that make sense, the other changes seem unnecessary.

Reply:
IMO all `/` should be escaped (rather than just some), to keep them all in the same format.

Reply:
If you tried to write `/^deps\/[^\/]+/` in our `lib/` folder, you'll get the following lint error:

```
error  Unnecessary escape character: \/                  no-useless-escape
```

Let's try to be consistent and not escape slashes unless it's needed.

Reply:
Oh, okay, I'll update the file later today.